app/editor/src/features/admin/reports/dashboard/utils/calcNextScheduleSend.ts

```diff
@@ -0,0 +1,89 @@
+import moment from 'moment';
+import { IReportModel, IReportScheduleModel } from 'tno-core';
+
+const WeekDays: Record<string, number> = {
```

Comment:
A lot of these should be stored in tno-core so that they are shared between apps.